zebra-rpc/Cargo.toml

```diff
@@ -42,6 +42,7 @@ jsonrpc-core = "18.0.0"
 jsonrpc-derive = "18.0.0"
 jsonrpc-http-server = "18.0.0"
 num_cpus = "1.15.0"
+regex = "1.7.1"
```

Comment:
Could we make this an optional dependency that's required by the "getblocktemplate-rpcs" feature if we keep it.

Reply:
Yea, i had been lazy here, sorry. Added it at https://github.com/ZcashFoundation/zebra/pull/6044/commits/377025fada23ad4275262d7cb3aeb692e9341825